.github/workflows/ci-arm64.yml

```diff
@@ -0,0 +1,79 @@
+name: Build OpenSSL Arm64
+
+on:
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+    inputs:
+      openssl_build:
+        description: 'OpenSSL Arm64'
```

Comment:
I'd suggest to keep inputs description consistent and state explicitly that this build the OpenSSL with MSVC.

Reply:
I would go further and explicitly specify, for each build, OS, architecture and toolchain.

Reply:
Changed